### ISSUE TITLE
Logic problem in override_asset_dir

### ISSUE THREAD

**Issue by MCMic** [smcameron/space-nerds-in-space#182]
There is this warning when building which makes perfect sense:

```
snis_client.c: In function 'override_asset_dir':
snis_client.c:19994:2: warning: this 'if' clause does not guard... [-Wmisleading-indentation]
  if (!d)
  ^~
snis_client.c:19996:3: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the 'if'
   return;
   ^~~~~~
```

And indeed if you look at snis_client.c you will see:

```
static void override_asset_dir(void)
{
	char *d;

	asset_dir = default_asset_dir;
	d = getenv("SNIS_ASSET_DIR");
	if (!d)
		printf("Assets will be loaded from %s (default)\n", default_asset_dir);
		return;
	printf("Assets will be loaded from %s (overriden)\n", d);
	asset_dir = d;
}
```

The code after the return will never be executed so it looks like this if is missing brackets.
I am not proposing a patch for this one because I do not know what this part of the code does, just that it *looks* wrong.

**Comment by smcameron** [comment]
I already fixed that a week ago: d509259b840ddd8aec79ad3231d05026afe51d83

:)

**Comment by MCMic** [comment]
Oops, sorry about that.
